xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -75,23 +75,23 @@ bool CGUIWindowPVRSearch::FindSimilar(const CFileItemPtr &item)
 
   // construct the search term
   if (item->IsEPG())
-    m_searchfilter.m_strSearchTerm = "\"" + item->GetEPGInfoTag()->Title() + "\"";
+    m_searchfilter.SetSearchTerm("\"" + item->GetEPGInfoTag()->Title() + "\"");
```

Comment:
... and there is one place where SetSearchTerm is called without that wrapping...